lua/entities/gmod_wire_expression2/core/hologram.lua

```diff
@@ -240,6 +240,7 @@ local function flush_clip_queue(queue, recipient)
 						net.WriteVector(clip.origin)
 						net.WriteFloat(clip.normal.x) net.WriteFloat(clip.normal.y) net.WriteFloat(clip.normal.z)
 						net.WriteBit(clip.isglobal ~= 0)
+						net.WriteUInt(clip.localentid, 16)
```

Comment:
Why do we sent the entity ID rather than just directly using [`net.WriteEntity`](https://wiki.garrysmod.com/page/net/WriteEntity)?

Reply:
The entity might not exist in clientside yet.